# This message represents data from the donkey rover
Header header

string  Scanner_Command		 # DoHoming, GoHome, Start, Stop
float64 Scanner_Ajustment_Angle  #rad -100 for no change
float64 Scanner_Roll_Angle       #rad -100 for no change
float64 Scanner_Home_Angle       #rad -100 for no change
float64 Scanner_Period           #sec -100 for no change
